File: untold_comparison_programs/03_functions/javascript/main.js
// Functions
const add = (a, b) => a + b;

const greet = (name, greeting = "Hello") => `${greeting}, ${name}!`;

const result = add(5, 3);
console.log(`5 + 3 = ${result}`);

const greeting = greet("Untold");
console.log(greeting);

const square = (x) => x * x;
console.log(`Square of 7: ${square(7)}`);

File: untold_comparison_programs/03_functions/python/main.py
# Functions
def add(a, b):
    return a + b

def greet(name, greeting="Hello"):
    return f"{greeting}, {name}!"

result = add(5, 3)
print(f"5 + 3 = {result}")

greeting = greet("Untold")
print(greeting)

square = lambda x: x * x
print(f"Square of 7: {square(7)}")